docs/reference/content/driver/reference/fle.md

```diff
@@ -0,0 +1,166 @@
++++
+date = "2019-06-13T09:00:01+01:00"
+title = "Field Level Encryption"
```

Comment:
We're using the term client side encryption now to emphasize the idea that the data is encrypted before leaving the client application.

Reply:
Done